specification/communication/data-plane/JobRouter/preview/2021-10-20-preview2/communicationservicejobrouter.json

```diff
@@ -0,0 +1,3568 @@
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Jobs"
+        ],
+        "summary": "Retrieves an existing job by Id",
+        "operationId": "JobRouter_GetJob",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "description": "Id of the job to retrieve",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/RouterJob"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a single Job": {
+            "$ref": "./examples/Jobs_GetJob.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Jobs"
+        ],
+        "summary": "Deletes a job and all of its traces.",
+        "operationId": "JobRouter_DeleteJob",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "description": "",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "204": {
+            "description": "Success"
+          }
+        },
+        "x-ms-examples": {
+          "Deletes a job and all of its traces": {
+            "$ref": "./examples/Jobs_DeleteJob.json"
+          }
+        }
+      }
+    },
+    "/routing/jobs/{id}/reclassify": {
+      "post": {
+        "tags": [
+          "Jobs"
+        ],
+        "summary": "Reclassify a job.",
+        "operationId": "JobRouter_ReclassifyJob",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "description": "Id of the job",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "reclassifyJobRequest",
+            "schema": {
+              "$ref": "#/definitions/ReclassifyJobRequest"
+            }
```

Comment:
The referenced schema has no properties, so it's not clear what should actually be sent in the body.  Is this even needed?